### DISCUSSION TITLE
The current runner images are fat, any chance to get an alpine based runner?

### DISCUSSION THREAD

**Discussion by bquenin** [actions/actions-runner-controller#1840] [Questions]
Hi there!

It seems that all the runner images are based on ubuntu and are therefore pretty big ~335 MB.

Is there any chance to see an Alpine based runner image? Or any other smaller distribution?

Thanks!

**Comment by grzesuav** [comment]
In my case `dive ghcr.io/actions/actions-runner:2.316.0` shows almost 1GB
```
Image name: ghcr.io/actions/actions-runner:2.316.0                                                                                                                                        -
Total Image size: 916 MB
```
@bquenin did you create an feature request for this ?

**Reply by grzesuav** [reply]
https://github.com/actions/actions-runner-controller/issues/3504 created one
